doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1045,56 +1045,62 @@ A subscription is represented by a JSON object with the following fields:
 * `id`: Subscription unique identifier. Automatically created at creation time.
 * `subject` : It s an object that describes the subject of the subscription.
 * `notification` : It is an object that describes the notification received by the subscriber.
-* `options` : It is a dictionary which can contain different parameters related to the subscription, including:
-  * `duration`: Duration of the subscription in ISO8601 format. Infinite if not specified.
+* `duration`: Duration of the subscription in ISO8601 format. Infinite if not specified.
 
-A 'subject' contains the following subfields:
+A `subject` contains the following subfields:
 
-* `entity` : Id or pattern id which identifies the affected entities (optional)
+* `id` : Id of the affected entities (optional). It cannot be use at the same time than `idPattern`.
+* `idPattern` : Id pattern of the affected entities (optional). It cannot be use at the same time than `id`.
 * `type` : Type of the affected entities (optional)
-* `attributes`: List of attributes to be monitored (if not specified it would mean all)
-* `condition`: Condition that will trigger the notification. It could be 'ONCHANGE' or 'ONTIMEINTERVAL'.
-By default will be ONCHANGE.
-* `expression`: Logical the expression on attributes. When its evaluation results in a
-`true` value the notification will be sent.
+* `condition`: Condition that will trigger the notification. It can have two optional properties, `attributes` (an array of
+attribute names) and `expression` (an expression composed of `q`, `geometry` and `coords`, see "List entities" operation above
+about this field):
+ * If `attributes` and `expression` are used, a notification is sent whenever one of the attributes in the `attributes` list changes and
+   at the same time `expression` matches.
+ * If `attributes` is used and `expression` is not used, a notification is sent whenever one of the attributes in the `attributes` list changes-
+ * If `attributes` is not used and `expression` is used, a notification is sent whenever any of the attributes of the entity changes and
+   at the same time `expression` matches.
+ * If neither `attributes` or `expression` are used, a notification is sent whenever any of the attributes of the entity changes.
 
-Entity or type must be present.
+`id`/`idPattern` or `type` must be present.
```

Comment:
Given that id/idPattern and type are described in different bullets, this the very first place in the text where I can describe restrictions applying both...